cloudfoundry-client-lib/src/main/java/org/cloudfoundry/client/lib/util/RequestOrdererV3.java

```diff
@@ -0,0 +1,32 @@
+package org.cloudfoundry.client.lib.util;
+
+import org.immutables.value.Value;
+
+@Value.Immutable
+public interface RequestOrdererV3 {
+    
+    String orderBy();
```

Comment:
Also, I'm concerned that this is a much lower level API than the rest of the CF Java client. I don't need to know the exact names of the query parameters in the other CF Java client methods, but here I do. Maybe we should at least have `String` constants for the valid values here?